README.md

```diff
@@ -20,8 +20,7 @@ pnpm i nest-commander @nestjs/common @nestjs/core
 
 ### CommandRunner
 
-Every command is seen as an `@Injectable()` by Nest, so your normal Dependency Injection still works as you would expect it to (woohoo!). The only thing to take note of is the interface `CommandRunner`, which should be implemented by each command. The `CommandRunner` interface ensures that all commands have a `run` method that return a `Promise<void>` and takes in the parameters `string[], Record<string, any>`. The `run` command is where you can kick all of your logic off from, it will take in whatever parameters did not match option flags and pass them in as an array, just in case you are really meaning to work with multiple parameters. As for the options, the `Record<string, any>`, the names of these properties match the `name` property given to the `@Option()` decorators, while their value matches the return of the option handler. If you'd like better type safety, you are welcome to create an interface for your options as well. You can view how the [Basic Command test](test/basic.command.ts) manages that if interested.
-
```

Comment:
This removed space won't mess up the markdown will it? Generally you should have a newline between all style changes (paragraph to heading, list to code, whatever it is)